### PR TITLE
Add crossborder plan

### PR DESCRIPTION
**Pull request by akashpinnaka**
I created crossborder plan which we planned earlier

### CODE REVIEW

**Comment by Rjgowda** [comment]
Add new line and some text

**Comment by akashpinnaka** [comment]
Ok

**Comment by Rjgowda** [comment]
rebase both commits into one

File: send_money/crossborder.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+THis is a file to handle cross border transactions.
+This is another line as ramesh suggested.
```
Comment (Rjgowda) [inline send_money/crossborder.txt:2]: looks good

**Review by Rjgowda — APPROVED** [review]

**Pull request merged**